services/media/service/src/domains/movies/plugins/all-movie-plugins.ts

```diff
@@ -22,7 +22,7 @@ export const AllMoviePlugins = makePluginByCombiningPlugins(
     'MoviesLicenseFilter',
   ),
   EntityListPublishingEndpointsPluginFactory({
-    id: 1,
+    id: -1,
```

Comment:
It seems like this change (and the one for tvshows) has broken some tests which
were expecting the ID to be 1. Is there an explicit need to set this to -1?
Would 0 work? (I'm fine with changing it, but not fond of the minus)

Reply:
Yes - started to change it to zero already on Friday but had some meetings - will finish that today.